examples/app_commands/basic.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+class MyBot(commands.Bot):
+
+    # In this basic example, we just synchronize the app commands to one guild,
+    # without requiring to set the guild to every command individually by just copying them over.
+    # By doing so we don't have to wait up to 1hr until they are shown for the end-user.
+
+    # If you want to synchronize your app commands to every guild your bot is in, 
+    # remove the following setup_hook and uncomment the one below.
+    async def setup_hook(self):
+        # Replace ... with the guild id you want your commands to be synced with.
+        guild = discord.Object(...)
+        self.tree.copy_global_to(guild=guild)
+        synced_app_commands = await self.tree.sync(guild=guild)
+        print(f'Synced the following commands to {guild.id}: {[c.name for c in synced_app_commands]}')
```

Comment:
I don't think there's much of a point to make use of the output here. Might as well leave it as a bare sync.